content/components/toggle-switch.mdx

```diff
@@ -0,0 +1,154 @@
+---
+title: Toggle switch
+description: A toggle switch is used to immediately toggle a setting on or off.
+---
+
+import {Box, Text} from '@primer/components'
+
+## Anatomy
+
```

Comment:
I would change the anatomy image to be full width, even if it's a small component, and add the list of components below it.